src/app/shared/components/template/components/layout/display-group/display-group.component.scss

```diff
@@ -89,3 +89,13 @@
     --border-color: var(--border-color-default, var(--ion-color-primary));
   }
 }
+
+.display-group-wrapper {
+  &[data-variant~="solid_secondary"] {
+    padding: var(--regular-padding);
```

Comment:
nit(blocking): I'm not sure it makes for just one variant to include padding by default whilst others do not (given that variants should also be able to stack/combine). I think likely better to either include as a deployment override, or add an additional variant that applies padding more consistently